curriculum/challenges/english/01-responsive-web-design/learn-css-colors-by-building-a-color-markers-set/step-014.md

```diff
@@ -13,10 +13,42 @@ In the `container` `div`, add two more `div` elements with the `class` `marker`.
 
 # --hints--
 
-Test 1
+Your first new `div` element should have an opening tag.
 
 ```js
+assert([...code.matchAll(/<div.*>/gi)][2]);
+```
+
+Your first new `div` element should have a closing tag.
+
+```js
+assert([...code.matchAll(/<\/div\s*>/gi)][2]);
+```
+
+Your second new `div` element should have an opening tag.
+
+```js
+assert([...code.matchAll(/<div.*>/gi)][3]);
+```
+
+Your second new `div` element should have a closing tag.
 
+```js
+assert([...code.matchAll(/<\/div\s*>/gi)][3]);
+```
+
+Your new `div` elements should be within the `container` `div` element.
```

Comment:
Small nitpick, because I have been trying to avoid two consecutive code blocks, and there is technically no `container` element - there is a `.container` element.

```suggestion
Your new `div` elements should be within the `div.container` element.
```

Reply:
Awesome, this looks good. I was thinking the same thing while writing the test text, and trying to come up with ways to refer to `div`s with certain class names while not implying that they're an HTML element.

This seems like the best way to do that, and learners should pick up on it quickly.

Once I finish writing the tests, I'll make another pass at the text and apply this method.